groups/nts/ntsu/ntsu_socketutil.cpp

```diff
@@ -1399,6 +1399,13 @@ ntsa::Error SocketUtil::send(ntsa::SendContext*       context,
 
     ssize_t sendmsgResult = ::sendmsg(socket, &msg, sendFlags);
 
+#if defined(BSLS_PLATFORM_OS_LINUX)
+    if (NTSCFG_UNLIKELY(sendmsgResult < 0 && errno == ENOBUFS)) {
```

Comment:
I think it should be rewritten as:
``` if (NTSCFG_UNLIKELY(sendmsgResult < 0 && errno == ENOBUFS && options.zeroCopy())) ```
The same is applicable to other send implementations.
The idea is to destinguish between ENOBUFS returned during ZC-send and ENOBUFS returned during notmal non-ZC-send

Reply:
Done